File: scripts/list_events.py
import sqlite3

conn = sqlite3.connect('autox.db')
cursor = conn.cursor()

print("=== ALL PHYSICAL EVENTS ===")
try:
    cursor.execute("SELECT id, title, start_date FROM physical_events")
    for row in cursor.fetchall():
        print(row)
except Exception as e:
    print(e)

conn.close()

File: scripts/list_events.js

import { getDb } from '../src/server/config/db.js';

const db = getDb();

const events = db.prepare("SELECT * FROM physical_events ORDER BY start_date").all();
events.forEach(e => {
    console.log(`[${e.id}] ${e.title} (${e.start_date}) Status: ${e.status}`);
});
